src/Microsoft.Azure.ServiceBus.MessageIdPlugin/Microsoft.Azure.ServiceBus.MessageIdPlugin.csproj

```diff
@@ -29,19 +26,9 @@
     <RootNamespace>Microsoft.Azure.ServiceBus.Plugins</RootNamespace>
     <GeneratePackageOnBuild>True</GeneratePackageOnBuild>
   </PropertyGroup>
-
-  <PropertyGroup Condition="'$(TargetFramework)' == 'uap10.0'">
-    <DefineConstants>$(DefineConstants);UAP10_0</DefineConstants>
-    <NugetTargetMoniker>UAP,Version=v10.0</NugetTargetMoniker>
-    <TargetPlatformIdentifier>UAP</TargetPlatformIdentifier>
-    <TargetPlatformVersion>10.0.14393.0</TargetPlatformVersion>
-    <TargetPlatformMinVersion>10.0.10240.0</TargetPlatformMinVersion>
-    <TargetFrameworkIdentifier>.NETCore</TargetFrameworkIdentifier>
-    <TargetFrameworkVersion>v5.0</TargetFrameworkVersion>
-  </PropertyGroup>
-  
+ 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.ServiceBus" Version="1.*" />
+    <PackageReference Include="Microsoft.Azure.ServiceBus" Version="2.*" />
   </ItemGroup>
   
   <ItemGroup Condition=" '$(TargetFramework)' == 'uap10.0' ">
```

Comment:
You could remove this `ItemGroup` as well

Reply:
This is the update to the `not required` comment.
Looks like some bug in the PR tool

Reply:
done